package pro.itfray.dto;

import java.util.UUID;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserProfileDto {

  private UUID uid;
  private String username;
  private String email;
  private String firstName;
  private String lastName;
  private String theme;
  private boolean hasOrganization;
}
